simple-facebook-instant-articles.php

```diff
@@ -178,21 +179,20 @@ public function setup_content_mods() {
 		add_shortcode( 'sigallery', array( $this, 'api_galleries_shortcode' ) );
 
 		// Render social embeds into FB IA format.
-		add_filter( 'embed_handler_html', array( $this, 'fb_formatted_social_embeds' ), 10, 3 );
-		add_filter( 'embed_oembed_html', array( $this, 'fb_formatted_social_embeds' ), 10, 4 );
-
-		// Render post content via DOM - to format it into FB IA format.
-		// DO it last, so content was altered via WP native hooks as much as possible.
-		add_filter( 'the_content', array( $this, 'fb_formatted_post_content' ), 1000 );
+		add_filter( 'embed_handler_html', array( $this, 'get_social_embed' ), 10, 3 );
+		add_filter( 'embed_oembed_html', array( $this, 'get_social_embed' ), 10, 4 );
```

Comment:
I'd call it `reformat_social_embed` - clearer and in line with other functions names.